data/operational.csv

```diff
@@ -0,0 +1,212 @@
+dateAdded,dateUpdated,uuid
```

Comment:
The [schema](https://github.com/alphagov/api-catalogue/blob/master/data/operational.csv-metadata.json) calls the "uuid" column "about", and it is the first column. Also "dateCreated" and "dateModified".